src/main/java/lotto/entity/AllLottoNumbers.java

```diff
@@ -0,0 +1,21 @@
+package lotto.entity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AllLottoNumbers {
+    private static final int MIN_LOTTO_NUMBER = 1;
+    private static final int MAX_LOTTO_NUMBER = 45;
+
+    private List<LottoNumber> lottoNumbers = new ArrayList<>();
```

Comment:
클래스를 분리하기 보다 캐싱을 목적으로 피드백 드렸었는데요
45개의 LottoNumber는 매번 인스턴스로 만들 필요가 없으니 클래스 변수로 재사용하는 방법도 고려해 보시면 좋을 것 같아요
```suggestion
    private static final List<LottoNumber> NUMBERS = new ArrayList<>();
```